CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Bug fixes
 
+* [#6383](https://github.com/rubocop-hq/rubocop/issues/6383): Add AllowBeforeTrailingComments option on Layout/ExtraSpacing cop. ([@davearonson][])
```

Comment:
Seems like a new feature to me, not a bugfix. :-)

Reply:
D'oh!  I coulda swore I had that in a feature section at some point.  Must have gotten moved around incorrectly during my resolution of the conflicts that arose from other stuff being merged in.  Sorry, fixed, will push soon.